server/server.go

```diff
@@ -0,0 +1,472 @@
+package server
+
+import (
+	"context"
+	"crypto/tls"
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"strings"
+	"sync"
+	"sync/atomic"
+	"syscall"
+	"time"
+
+	epplib "github.com/dotse/epp-lib"
+)
+
+type errorLog interface {
```

Comment:
Perhaps we can stick with one logger interface?

Reply:
Sure!